### PR TITLE
[elyses-enchantments] Be super explicit about mutation+return

### PR DESCRIPTION
**Pull request by iHiD**
As some people are getting stuck/confused by the fact that the functions should both mutate and return, let's be super-specific about it.

### CODE REVIEW

File: exercises/concept/elyses-enchantments/.docs/instructions.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ of a certain card corresponds to the index in the array. That means
 position 0 refers to the first card, position 1 to the second card
 etc.
 
+_Note: All functions should update the array of cards and then return the modified array - a common way of working known as the Builder pattern, which allows you to nicely daisy-chain functions together._
```
Comment (iHiD) [inline exercises/concept/elyses-enchantments/.docs/instructions.md:12]:
```suggestion
~~~~exercism/note
All functions should update the array of cards and then return the modified array - a common way of working known as the Builder pattern, which allows you to nicely daisy-chain functions together.
````
```

**Review by SleeplessByte — APPROVED** [review]
haha, "what's a daisy chain"

**Comment by SleeplessByte** [comment]
/format

you're code ugly @iHiD

**Comment by iHiD** [comment]
> haha, "what's a daisy chain"

https://ajnikrant.medium.com/method-chaining-or-daisy-chaining-in-ruby-31713b75ef30

There's probably better words/phrases for non-method world

**Comment by SleeplessByte** [comment]
pretty admonition

it pretty

Oh yeah, I know what it is ;)

**Pull request merged**